__tests__/components/NetworkSwitcher.test.js

```diff
@@ -31,21 +31,21 @@ describe('NetworkSwitch', () => {
   test('correctly renders MainNet initially', () => {
     const { wrapper } = setup()
 
-    const networkSelectorElement = wrapper.find('select').getElement()
+    const networkSelectorElement = wrapper.find('button[data-value="MainNet"]')
 
-    expect(networkSelectorElement.props.defaultValue).toEqual('MainNet')
+    expect(networkSelectorElement.children().length).toBe(3)
   })
 
   test('switches to the correct network when chosen from the dropdown', async () => {
     const { wrapper } = setup()
 
     const instance = wrapper.instance()
-    const networkSelector = wrapper.find('select')
-    networkSelector.simulate('change', { target: { value: 'TestNet' } })
+    const networkSelector = wrapper.find('#networkDropdownMenu')
```

Comment:
Do we definitely need to do this based on ID? I think this component should always only have one select box right?

Reply:
I'm not sure. The wrapper is targeting a div that handles event propagation from multiple buttons. The classname can not be targeted, because CSS modules create random class names on build. Maybe we can use a wildcard selector like: [class*='networkDropDown']